examples/amqp/src/main/java/io/cloudevents/examples/amqp/vertx/AmqpClient.java

```diff
@@ -0,0 +1,160 @@
+ */
+public class AmqpClient {
+
+    private static ProtonConnection connection;
+
+    private static final String SERVER_HOST = "127.0.0.1";
+    private static final int SERVER_PORT = 5672;
+    private static final String SEND_MESSAGE = "send";
+    private static final String RECEIVE_MESSAGE = "receive";
+
+    private static PrintWriter writer = new PrintWriter(System.out, true);
+
+    public static void main(String args[]) {
+
+        if (args.length < 1) {
+            writer.println("Usage: AmqpClient [send|receive]");
+            return;
+        }
+
+        final String action = args[0].toLowerCase();
+
+        final Vertx vertx = Vertx.vertx();
+
+        switch (action) {
+
+        case SEND_MESSAGE:
+            connectToServer(vertx, SERVER_HOST, SERVER_PORT)
+                    .compose(conn -> {
+                        connection = conn;
+                        writer.printf("[Client] Connected to %s:%s", SERVER_HOST, SERVER_PORT);
+
+                        return openSenderLink();
+                    }).map(sender -> {
+
+                        final JsonObject payload = new JsonObject().put("temp", 50);
+                        final String to = "/telemetry";
+
+                        final Message message = ProtonHelper.message(to, payload.toString());
+
+                        // set attributes
+                        final Map<String, Object> attributes = new HashMap<>();
+                        attributes.put("cloudEvents:type", "com.example.sampletype1");
+                        attributes.put("cloudEvents:source", "http://127.0.0.1/amqp-client");
+                        attributes.put("cloudEvents:id", "client-id");
+                        attributes.put("cloudEvents:specversion", "1.0");
+                        attributes.put("cloudEvents:time", "2020-11-06T21:47:12.037467+00:00");
+                        message.setApplicationProperties(new ApplicationProperties(attributes));
+
+                        sender.send(message, delivery -> {
+                            if (Accepted.class.isInstance(delivery.getRemoteState())) {
+                                writer.println("[Client:] message delivered and accepted by remote peer");
+                            }
+                            connection.close();
+                        });
+                        return null;
+                    }).otherwise(t -> {
+                        writer.printf("[Client] Connection failed (%s)", t.getCause().getMessage()); 
+                        return null;
+                    });
+            break;
+        case RECEIVE_MESSAGE:
+            connectToServer(vertx, SERVER_HOST, SERVER_PORT)
+                    .compose(conn -> {
+                        connection = conn;
+                        writer.println("[Client] Connected");
+                        return Future.succeededFuture();
+                    }).map(success -> {
```

Comment:
This lambda has only one stmt (the return), collapse it without the {} brackets and return stmt